.github/workflows/nodejs.yml

```diff
@@ -30,10 +30,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 1
-      - name: Use Node.js 12.14.1
+      - name: Use Node.js 14.18.X
         uses: actions/setup-node@v2
         with:
-          node-version: 12.14.1
+          node-version: 14.18
```

Comment:
Think it should be okay to not be specific about patch version but happy to change if necessary